core-web/apps/dotcms-block-editor/src/app/app.module.ts

```diff
@@ -24,9 +29,10 @@ import { AppComponent } from './app.component';
         BlockEditorModule,
         OrderListModule,
         ListboxModule,
-        HttpClientModule
+        HttpClientModule,
+        DotAssetSearchComponent
     ],
-    providers: [DotPropertiesService]
+    providers: [DotPropertiesService, DotContentSearchService, DotLanguagesService]
```

Comment:
We trying to have same instance of this service for the whole app, why?

Reply:
Since we have moved these services to the `ui` library, we need to import them into `app.module.ts` (the Block Editor application) so that the web component does not throw `service not injected` errors.